test/gtest/uct/test_md.cc

```diff
@@ -653,6 +653,10 @@ UCS_TEST_SKIP_COND_P(test_md, invalidate, !check_caps(UCT_MD_FLAG_INVALIDATE))
     ucs_status_t status;
     uct_md_mem_dereg_params_t params;
 
+    if (GetParam().md_name == "cuda_ipc") {
```

Comment:
how about using has_cuda_ipc() ?

Reply:
Noticed that has_cuda_ipc is a virtual defined in uct_test class which is not accessible in this function.

Reply:
ok, test_md does not inherit from uct_test